### ISSUE TITLE
invoke service not support route '/'

### ISSUE THREAD

**Issue by rrylee** [dapr/dapr#2571]
## In what area(s)?

> /area runtime

## What version of Dapr?

1.0.0-rc.1

## Expected Behavior

my service have route '/'，dapr invoke service can call '/' method

## Actual Behavior

dapr will return ERR_DIRECT_INVOKE error

## Steps to Reproduce the Problem

1. `dapr run --app-id=my-service` a http service, http mux handler '/' route.

2. use go-sdk invoke my-service

```go
import (
	"context"
	"fmt"

	dapr "github.com/dapr/go-sdk/client"
)

func main() {
	client, err := dapr.NewClient()
	if err != nil {
		panic(err)
	}
	defer client.Close()

	content := &dapr.DataContent{}
	client.InvokeMethodWithContent(context.Background(), "my-service", "/", "post", content) // will return err
}
```

## Release Note

RELEASE NOTE: FIX can not invoke service http route '/'

**Comment by yaron2** [comment]
Closed with #2568

**Comment by yaron2** [comment]
Thank you @rrylee
